crates/sc-consensus-subspace-rpc/src/lib.rs

```diff
@@ -212,27 +120,27 @@ pub trait SubspaceRpcApi {
 #[derive(Default)]
 struct ResponseSenders {
     current_slot: Slot,
-    senders: Vec<async_oneshot::Sender<ProposedProofOfReplicationResult>>,
+    senders: Vec<async_oneshot::Sender<ProofOfReplication>>,
 }
 
-/// Implements the SubspaceRpc trait for interacting with Subspace.
-pub struct SubspaceRpcHandler<Block, Client> {
+/// Implements the [`SubspaceApi`] trait for interacting with Subspace.
+pub struct Subspace<Block, Client> {
```

Comment:
`Subspace` is too generic name for a struct, `SubspaceRpcHandler` was a much more specific name, why changing it?

Reply:
same above.